ister_gui.py

```diff
@@ -644,6 +672,247 @@ def validate_condition(self):
         return True
 
 
+class ChooseAction(ProcessStep):
+    """UI to select installation path (Install, Shell, or Repair)"""
+    def __init__(self):
+        super(ChooseAction, self).__init__()
+        self.choices = {
+            'Install': 'Install          - Install Clear Linux OS',
+            'Repair': 'Repair           - Attempt to repair host OS',
+            'Shell': 'Shell            - Open shell in host OS',
+            'Update Installer': 'Update Installer - Update installer to '
```

Comment:
If it goes off the rails they can always fall back to that. Agree it's a bit different - seems to work just fine though.